src/components/tutorials/advanced-tutorial.js

```diff
@@ -96,18 +95,16 @@ class AdvancedTutorial extends React.Component {
                 </tbody>
             } />
             <Advanced2 />
-            <p>A <code>201 Created</code> - great. We'll look at what ACI scores look like next:</p>
-            <h2 id='aci-scoring'>
+            <p>A <code>201 Created</code> - great. We'll look at what the score for the updated submission record looks like next:</p>
+            <h2 id='updated-scoring'>
               <a
                 className='tutorial-header-link'
-                href='#aci-scoring'>
-              ACI Scoring
+                href='#updated-scoring'>
+                Get a score for the submission record with the updated measurement set
             </a>
             </h2>
-            <p>It's possible that a different measurement set added to this submission could score differently, for example.</p>
             <p>For submissions with more performance data, this breakdown gives us visibility
```

Comment:
update paragraph to: 

<p>For submissions with more performance data, this breakdown gives us visibility into how the individual measurements contribute to the aggregate. The additional visibility helps because sometimes a submission can be valid, but incomplete for scoring purposes.</p>